apps/cms/src/app/(auth)/settings/visual-identity/utils/config.ts

```diff
@@ -13,23 +12,35 @@ export type VisualIdentityFormData = Pick<
 export function normalizeVisualIdentityFormData(
     data: VisualIdentityFormData,
 ): CompanyUpdateInput {
-    return {
-        ...omit(data, ['logoMedia', 'favIconMedia']),
+    const result: CompanyUpdateInput = {}
+
+    if (typeof data.logo !== 'undefined') {
+        result.logo = data.logo || null
+    }
+
+    if (typeof data.favIcon !== 'undefined') {
+        result.favIcon = data.favIcon || null
     }
+
+    if (typeof data.settings !== 'undefined') {
+        result.settings = data.settings
+    }
+
+    return result
```

Comment:
```suggestion
     return {
         logo: data.logo ?? null
         favIcon: data.favIcon ?? null
         settings: data.settings || undefined
     }
```

O que você fez funciona exatamente igual, só da maneira que estou sugerindo é menos verboso. Mas se não quiser não precisar alterar.